homeassistant/package_constraints.txt

```diff
@@ -46,7 +46,6 @@ ifaddr==0.2.0
 Jinja2==3.1.6
 lru-dict==1.3.0
 mutagen==1.47.0
-numpy==2.3.0
```

Comment:
I think we should still keep this to ensure we constrain numpy to the version required by the built-in camera integration via stream